README.md

```diff
@@ -57,19 +57,18 @@ On release, automated continuous integration tests run the pipeline on a full-si
 
 4. Start running your own analysis!
 
-   The pipeline takes care of downloading the required databases and parameters required by AlphaFold2 and/or Colabfold. In case you have already downloaded the required files, you can skip this step by providing the path using the corresponding parameter [`--af2_db`] or [`--colabfold_db`]
+   The pipeline takes care of downloading the required databases and parameters required by AlphaFold2 and/or Colabfold. In case you have already downloaded the required files, you can skip this step by providing the path using the corresponding parameter [`--alphafold2_db`] or [`--colabfold_db`]
 
 - Typical command to run AlphaFold2 mode:
```

Comment:
Would it make sense to put also the command to run the AlphaFold2 split mode in the docs, since we do so for colabfold submodes?

Reply:
Fixed here [9b9d109](https://github.com/nf-core/proteinfold/pull/61/commits/9b9d109804703b5ed2c2972a9c8de4c723bfb67c)